app/assets/javascripts/analytics-ga4/ga4-finder-tracker.js

```diff
@@ -83,6 +86,27 @@
             wasFilterRemoved = true
           }
           break
+
+        case 'date': {
```

Comment:
Little bit eccentric yeah – Standard won't let you define new variables inside a case without an explicit scope (as it would leak otherwise), so the `const inputs` below would fail linting.

Reply:
oh that's a bit annoying! thanks for explaining - fine to leave in.